src/components/Messages/Message/index.js

```diff
@@ -83,7 +91,9 @@ const renderContent = ({
 		),
 	text && (
 		<MessageBubble inverse={me} msgSequence={msgSequence} quoted={quoted} system={system}>
-			<MessageText text={text} system={system} />
+			{showPostChatUrl && <MessageText className = {createClassName(styles, 'surveyText__content')} system={system} text={'Thank you for chatting with us! How did we do today?'} />}
```

Comment:
Was this part of the requirement? I couldn't find it. IMO it's better to just use the `Chat Ended` message here. So that we can also configure it as per need

Reply:
Yes, it was part of the design posted in the issue.

Reply:
@bhardwajaditya We just discussed it in the meeting to make it configurable. So for now we can configure the `Chat ended` message with this thank you message. So no need to add static text here.

Sorry about the requirement change. Can you please remove it?  Thanks!

Reply:
@Shailesh351 it's not clear to me, we need to keep it or remove it entirely?

Reply:
@bhardwajaditya Sorry for the confusion. We decided to remove it. Because the `Chat ended` message is configurable and we can replace it with `Thank you for chatting with us!`.